mdp/nodes/sfa_nodes.py

```diff
@@ -435,7 +606,7 @@ def _train(self, x, include_last_sample=None):
 
     def _set_range(self):
         if (self.output_dim is not None) and (
-            self.output_dim <= self._expnode.output_dim):
+                self.output_dim <= self._expnode.output_dim):
```

Comment:
like mentioned earlier, please restore the additional indent of continuation lines.